org.wxhexeditor.wxHexEditor.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "org.wxhexeditor.wxHexEditor",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "wxHexEditor",
+    "rename-icon": "wxHexEditor",
+    "rename-desktop-file": "wxHexEditor.desktop",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=all"
+    ],
+    "modules": [
+        {
+          "name": "wxWidgets",
+          "cleanup": [
+              "/bin",
+              "/include",
+              "/share/bakefile",
+              "/share/aclocal",
+              "/lib/wx/include"
+          ],
+          "config-opts": [
+              "--with-gtk=3",
+              "--disable-sdltest",
+              "--enable-unicode",
+              "--enable-display",
+              "--enable-propgrid",
+              "--disable-webkit",
+              "--disable-webview",
+              "--disable-webviewwebkit",
+              "--with-expat=builtin",
+              "--with-libiconv=/usr"
+          ],
+          "build-options" : {
+              "cxxflags":"-std=c++0x"
+          },
+          "sources": [ 
+            {  
+                "type": "archive", 
+                "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
+                "sha256":"96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+            }
+          ]
+        },
+        {
+            "name": "wxHexEditor",
+            "buildsystem": "simple",
```

Comment:
Why so? Why not the default `autotools`? You can always install icons and other files in the `post-install` section.

Reply:
Basically why not... with all the renaming and shuffling around of resources i figured its better to just do the install myself until i maybe get it sorted out upstream.

Reply:
Well, you should use `autotools` (or `cmake-ninja` or `meson`) whenever possible.
For example, it will take care of parallel `make`. As far as I know, it should work for wxHexEditor. If you decide to use the `simple` buildsystem, you will have to add `-j"$(nproc)"` manually (unless it doesn't support it).

Reply:
You're right, for this version ill keep it this manual - not very pretty - way.
But as i see this as an upstream issue i will try to fix it there instead.

Reply:
I wasn't talking about upstream changes, but about this manifest.
Why don't you use `autotools` in flatpak to provide parallel make? Have you even try to build this in parallel?
https://github.com/frhun/flathub/blob/a80da4d5580162e91c6ea4df0a90bcff921672e5/org.wxhexeditor.wxHexEditor.json#L52